pkg/log/log.go

```diff
@@ -36,26 +38,44 @@ type GlobalConfig struct {
 }
 
 func init() {
-	zapCfg := zap.NewDevelopmentConfig()
-	zapCfg.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-	zapCfg.Level.SetLevel(zap.InfoLevel)
+	zapCfg := newZapConfigWithInfoLevel()
 	l, err := zapCfg.Build()
 	if err != nil {
 		log.Println("Failed to init zap global logger, no zap log will be shown till zap is properly initialized: ", err)
 		return
 	}
+	consensusZapCfg := newZapConfigWithInfoLevel()
+	_consensusLogger, err = consensusZapCfg.Build()
```

Comment:
seems like there is no way to set configuration through yaml for this logger. we cannot direct it to s.log and set format to json.